cmd/sites.go

```diff
@@ -0,0 +1,158 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"text/tabwriter"
+
+	"github.com/joinflux/webflowctl/internal"
```

Comment:
Interesting, with this import here, if you changed something in `internal`, do the changes propagate?

Or do you have something equivalent to `yarn link`?

Reply:
This is just how modules work in go. This import is still local and you can find this in the `internal` folder at the root of the repo. I have been thinking of changing it to `pkg` tho but it's just convention. `internal` is meant to be something that is not consumed externally which is the case for the Webflow API Client.

Reply:
Ah, it's because of `module github.com/joinflux/webflowctl` in `mod.go`?

Reply:
That's correct. From the [spec/docs](https://go.dev/ref/mod#glos-module-path): 

> module path: A path that identifies a module and acts as a prefix for package import paths within the module. For example, "golang.org/x/net".